scenarios/k8s-scaling/cm.yml

```diff
@@ -0,0 +1,97 @@
+apiVersion: v1
```

Comment:
Isn't it a duplicate of rules.drl? If so remove this file please

Reply:
it is but when you actualy want to try it out it's mych easier to just run `kubectl apply -f cm.yaml ` then coping it yourself

Reply:
```
make upload-rules SCENARIO_NAME=k8s-scaling
```
does exactly that, no need to keep this file

Reply:
ehhh, okeeeey then